tests/testthat/test-wflow_start.R

```diff
@@ -389,6 +389,16 @@ test_that("wflow_start changes to workflowr directory by default", {
   expect_identical(getwd(), site_dir)
 })
 
+test_that("wflow_start fails if `overwrite = TRUE` and `existing = FALSE`", {
+
+  site_dir <- tempfile("test-start-")
+  site_dir <- workflowr:::absolute(site_dir)
+  expect_error(wflow_start(site_dir, change_wd = FALSE, overwrite = TRUE,
+                           user.name = "Test Name", user.email = "test@email"),
+               "Cannot overwrite non-existent project. Set existing = TRUE if you wish to overwrite existing workflowr files.")
```

Comment:
Could you please shorten the string that the test expects? That will avoid the potential console width issue we discussed earlier.
```suggestion
               "Cannot overwrite non-existent project.")
```